ports/servoshell/prefs.rs

```diff
@@ -669,11 +669,7 @@ pub(crate) fn parse_command_line_arguments(args: Vec<String>) -> ArgumentParsing
         userscripts_directory: cmd_args.userscripts,
         experimental_prefs_enabled: cmd_args.enable_experimental_web_platform_features,
         #[cfg(target_env = "ohos")]
-        log_filter: Some(
-            cmd_args
-                .log_filter
-                .unwrap_or(preferences.log_filter.clone()),
-        ),
+        log_filter: cmd_args.log_filter.or(Some(preferences.log_filter.clone())),
```

Comment:
I think you still need to check here if `references.log_filter` is empty and map that None.